lms/templates/navigation.html

```diff
@@ -42,7 +42,7 @@
     <h1 class="logo">
       <a href="${marketing_link('ROOT')}">
         <%block name="navigation_logo">
-            <img src="${static.url(branding.get_logo_url())}" alt="${platform_name()}"/>
+            <img src="${static.url(branding.get_logo_url())}" alt="${platform_name()} ${_('Home Page')}"/>
```

Comment:
Please put platform_name as part of the i18n string - in other languages, the order of the words may change (eg "page of home of platform_name")

See https://github.com/edx/edx-platform/blob/master/lms/templates/certificates/_accomplishment-header.html#L8